src/analyticsEnum.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * This is an enum that stores all the
+ * possible category names and associated actions
+ * for Google Analytics
+ */
+const analyticsEnum = {
```

Comment:
Let's rename this file and this object to just `analytics`. Sounds cleaner.
```suggestion
const analytics = {
```

Reply:
Agreed